### PR TITLE
pod-security policy charts update

### PR DESCRIPTION
**Pull request by dolisss**

### CODE REVIEW

File: charts/pod-security-policies/templates/disallow-privileged-containers.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        Privileged mode is disallowed. The fields spec.containers[*].securityContext.privileged
+        and spec.initContainers[*].securityContext.privileged must not be set to true.
+      pattern:
+        spec:
+          =(initContainers):
+          - =(securityContext):
+              =(privileged): "false"          
+          containers:
+          - =(securityContext):
+              =(privileged): "false"
```
Comment (pns-nirmata) [inline charts/pod-security-policies/templates/disallow-privileged-containers.yaml:29]: no newline

File: charts/pod-security-policies/templates/restrict-sysctls.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        Setting additional sysctls above the allowed type is disallowed.
+        The field spec.securityContext.sysctls must not use any other names
+        than 'kernel.shm_rmid_forced', 'net.ipv4.ip_local_port_range',
+        'net.ipv4.tcp_syncookies' and 'net.ipv4.ping_group_range'.
+      pattern:
+        spec:
+          =(securityContext):
+            =(sysctls):
+            - name: "kernel.shm_rmid_forced | net.ipv4.ip_local_port_range | net.ipv4.tcp_syncookies | net.ipv4.ping_group_range"
+              value: "?*"
```
Comment (pns-nirmata) [inline charts/pod-security-policies/templates/restrict-sysctls.yaml:32]: no newline

**Review by pns-nirmata — APPROVED** [review]
Some errors around newlines. Ok once those are fixed.

File: charts/pod-security-policies/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: pod-security-policies
+description: Pos Security Policy Set
```
Comment (atulatnirmata) [inline charts/pod-security-policies/Chart.yaml:3]: Is this a typo? Pos -> Pod

File: charts/pod-security-policies/values.yaml [hunk 4]
```diff
@@ -0,0 +1 @@
+# not parameters needed for now.
```
Comment (atulatnirmata) [inline charts/pod-security-policies/values.yaml:1]: typo? not->no

File: charts/pod-security-policies/Chart.yaml (hunk 3 above)
Comment (dolisss) [reply charts/pod-security-policies/Chart.yaml]: Yes, Updated. Please check.

File: charts/pod-security-policies/values.yaml (hunk 4 above)
Comment (dolisss) [reply charts/pod-security-policies/values.yaml]: Yes, Updated now.  Please check.

**Review by atulatnirmata — APPROVED** [review]

**Pull request merged**